ui/app/AppLayouts/Chat/panels/communities/ChatPermissionQualificationPanel.qml

```diff
@@ -0,0 +1,123 @@
+        anchors.fill: parent
+        spacing: d.panelRowSpacing
+        StatusBaseText {
+            text: qsTr("To post, hold")
+            font.pixelSize: Style.current.primaryTextFontSize
+            color: Style.current.secondaryText
+        }
+
+        StatusScrollView {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 20
+            contentWidth: tokenRow.implicitWidth
+            padding: 0
+            Row {
+                id: tokenRow
+                Repeater {
+                    id: repeater
+                    model: root.viewOnlyHoldingsModel
+                    Row {
+                        spacing: d.panelRowSpacing
+                        Row {
+                            spacing: d.panelRowSpacing
+                            Repeater {
+                                model: HoldingsSelectionModel {
+                                    sourceModel: holdingsListModel
+                                    assetsModel: root.assetsModel
+                                    collectiblesModel: root.collectiblesModel
+                                }
+                                StatusListItemTag {
+                                    height: 20
+                                    enabled: false
+                                    leftPadding: 2
+                                    title: model.text
+                                    asset.name: model.imageSource
+                                    asset.isImage: true
+                                    asset.bgColor: "transparent"
+                                    asset.height: 16
+                                    asset.width: asset.height
+                                    asset.bgWidth: asset.height
+                                    asset.bgHeight: asset.height
+                                    asset.color: asset.isImage ? "transparent" : titleText.color
+                                    closeButtonVisible: false
+                                    titleText.color: model.available ? Theme.palette.primaryColor1 : Theme.palette.dangerColor1
+                                    bgColor: model.available ? Theme.palette.primaryColor2 :Theme.palette.dangerColor2
+                                    titleText.font.pixelSize: 12
+                                }
+                            }
+                        }
+                        StatusBaseText {
+                            height: parent.height
+                            visible: (index !== (repeater.count - 1))
+                            verticalAlignment: Text.AlignVCenter
+                            font.pixelSize: Style.current.primaryTextFontSize
+                            rightPadding: d.panelRowSpacing
+                            color: Style.current.secondaryText
+                            text: qsTr("or")
+                        }
+                    }
+                }
+            }
+        }
+
+        ToolButton {
```

Comment:
I'm wondering if it would be better to have the mouse hand shape when overing on this region.. if not, it's difficult to understand the area is clickable. WDYT?

Reply:
done